e2e/cypress/integration/integrations/incoming_webhook/webhook_posts_when_creator_not_in_channel_spec.js

```diff
@@ -0,0 +1,97 @@
+
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+import {getRandomId} from '../../../utils';
+
+describe('Integrations', () => {
+    let testUser;
+    let secondUser;
+    let testTeam;
+    let testChannel;
+    let newIncomingHook;
+    let incomingWebhook;
+
+    before(() => {
+        // # Create new setup
+        cy.apiInitSetup().then(({user}) => {
+            testUser = user;
+
+            // # Creaate a second user
+            cy.apiCreateUser().then(({user: user2}) => {
+                secondUser = user2;
+            });
+
+            // # Login as the new user
+            cy.apiLogin(testUser).then(() => {
+                // # Create a new team with the new user
+                cy.apiCreateTeam('test-team', 'Team Testers').then(({team}) => {
+                    testTeam = team;
+
+                    // # Add second user to the test team
+                    cy.apiAddUserToTeam(testTeam.id, secondUser.id);
+
+                    // # Create a new test channel for the team
+                    cy.apiCreateChannel(testTeam.id, 'test-channel', 'Testers Channel').then(({channel}) => {
+                        testChannel = channel;
+
+                        newIncomingHook = {
```

Comment:
If not to be used elsewhere, then declare within this scope
```suggestion
                        const newIncomingHook = {
```